lib/tuya.js

```diff
@@ -1523,6 +1499,19 @@ const valueConverter = {
     },
 };
 
+const valueConverterAdvance = {
+    thermostatScheduleDayMultiDPWithDayNumber: (dayNum) => {
```

Comment:
I don't believe it is necessary to put this in `valueConverterAdvance`, it should work fine in `valueConverter` since `valueConverter` is only referenced in the lambdas.

Reply:
will try